django/contrib/postgres/constraints.py

```diff
@@ -0,0 +1,82 @@
+from django.db.models import F
+from django.db.models.constraints import BaseConstraint
+from django.db.models.sql import Query
+
+__all__ = ('ExclusionConstraint',),
```

Comment:
`flake8plus` checks for tuples vs. lists (probably because a tuple is immutable, and thus slightly faster) with `__all__`. I'm open to both, however, as the current style appears to be a list.

Reply:
Ok. Either way we need to lose the trailing comma.